tests/FacebookTest.php

```diff
@@ -177,7 +177,7 @@ public function testPersistentDataHandlerCanBeForced()
     }
 
     /**
-     * @expectedException \InvalidArgumentException
+     * @expectedException PHPUnit_Framework_Error
```

Comment:
Just a clarification; failed argument parsing throws a [`TypeError`](http://php.net/typeerror) in PHP 7. [See example](https://3v4l.org/IYvad). :)